polly/test/Support/dumpfunction.ll

```diff
@@ -1,9 +1,9 @@
 ; New pass manager
-; RUN: opt %loadNPMPolly -O3 -polly -polly-position=before-vectorizer -polly-dump-before --disable-output %s
+; RUN: opt %loadPolly -O3 -polly -polly-position=before-vectorizer -polly-dump-before --disable-output %s
 ; RUN: FileCheck --input-file=dumpfunction-callee-before.ll --check-prefix=CHECK --check-prefix=CALLEE %s
 ; RUN: FileCheck --input-file=dumpfunction-caller-before.ll --check-prefix=CHECK --check-prefix=CALLER %s
 ;
-; RUN: opt %loadNPMPolly -O3 -polly -polly-position=before-vectorizer -polly-dump-after --disable-output %s
+; RUN: opt %loadPolly -O3 -polly -polly-position=before-vectorizer -polly-dump-after --disable-output %s
 ; RUN: FileCheck --input-file=dumpfunction-callee-after.ll --check-prefix=CHECK --check-prefix=CALLEE %s
 ; RUN: FileCheck --input-file=dumpfunction-caller-after.ll --check-prefix=CHECK --check-prefix=CALLER %s
```

Comment:
These lines are identical to the ones above and can be removed. They were originally meant to test the legacy pass manager but the difference disappeared in 6c0fe14414cd0b3cbd96e809843ebbe3428da64e.

Reply:
These seem to be testing the polly dumps before and after and they both seem to be quite different. I think I have removed all the duplicate lines in tests that were testing both NPM and LPM passes previously (there were a few like this)

Reply:
Correct, I seem to have looked for where the legacy pass tests are but git blame doesn't show them anymore (since remove). Thanks for clearing up.